package.json

```diff
@@ -7,10 +7,11 @@
     "npm": ">=3.8"
   },
   "scripts": {
-    "start": "webpack-dev-server --config tools/webpack.config.babel.js --progress --env.prod=false --env.config_name=integreat-debug",
-    "build": "webpack --config tools/webpack.config.babel.js --env.prod=true --env.config_name=integreat",
-    "build:debug": "webpack --config tools/webpack.config.babel.js --env.prod=false --env.config_name=integreat-debug",
-    "build:lib": "webpack --config tools/lib.webpack.config.babel.js --env.prod=true && flow-copy-source -i '**/@(__mocks__|__tests__)/**' ./src ./lib-dist && mv ./lib-dist/lib.js.flow ./lib-dist/index.js.flow",
+    "start": "webpack-dev-server --config build/webpack.config.babel.js --progress --env.config_name=integreat-debug",
+    "start:malte": "webpack-dev-server --config build/webpack.config.babel.js --progress --env.config_name=malte",
+    "build": "webpack --config build/webpack.config.babel.js --env.config_name=integreat",
```

Comment:
I think it is easier to understand if we keep `--env.prod=false` here. That way determining whether you just created a production build or not is just a single "hop" away